### ISSUE TITLE
Set control type to Keyboard and Joystick by default

### ISSUE THREAD

**Issue by RibShark** [RibShark/TonicTroubleFix#9]
Right now the game defaults to Keyboard only which is confusing for people who have gamepads. Control type should be set to Keyboard and Joystick by default ideally.
